fastq/phredQual.go

```diff
@@ -16,18 +18,31 @@ func ToQualUint8(qual []rune) []uint8 {
 	return answer
 }
 
+// ToQual will convert a slice of bytes to uint8 and subtracting a 33 offset to the values
+func ToQual(qual []byte) []uint8 {
+	var answer []uint8 = make([]uint8, len(qual))
+	for i := 0; i < len(qual); i++ {
+		answer[i] = uint8(qual[i]) - asciiOffset
```

Comment:
I don't think you need this conversion.  I believe that "byte" is not an actual type by itself.  I believe it is just an alternative name that can be used for uint8.  I believe "bytes" is just for humans and the compiler will see it as "uint8".